### PR TITLE
Ready

### PR DESCRIPTION
**Pull request by OSCAR-WOS**
**Please describe the changes this PR makes and why it should be merged:**
Please set a better PR title.

**Status**

- [ ] Code changes have been tested against the Discord API, or there are no code changes

**Semantic versioning classification:**

- [ ] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: src/api/controllers/misc.ts [hunk 1]
```diff
@@ -27,11 +27,31 @@ export async function handleInternalReady(
...
+  cache.shardedGuilds += payload.guilds.length;
   eventHandlers.shardReady?.(shardID);
+
+  //Ability to fine tune
+  const chunk = 1200;
+  const buffer = 400;
+  const request = 7;
+
+  //Remaining guilds to load via handleInternalGuildCreate
+  function checkRemaining(left: any) {
```
Comment (itohatweb) [inline src/api/controllers/misc.ts:39]: Try finding a corresponding type and not any,

**Review by itohatweb — CHANGES_REQUESTED** [review]

**Review by Skillz4Killz — CHANGES_REQUESTED** [review]
This has the same issues as the other PR i reviewed.

File: src/api/controllers/misc.ts (hunk 1 above)
Comment (ayntee) [reply src/api/controllers/misc.ts:39]: `number`

**Comment by ayntee** [comment]
Feel free to un-draft this PR when the suggested changes are applied.

File: src/api/controllers/misc.ts [hunk 2]
```diff
@@ -30,24 +30,18 @@ export async function handleInternalReady(
...
-  const chunk = 1200;
-  const buffer = 400;
-  const request = 7;
-
-  //Remaining guilds to load via handleInternalGuildCreate
-  function checkRemaining(left: any) {
-    //If higher than chunk, delete chunk value, otherwise set to 0
-    cache.shardedGuilds >= chunk
-      ? cache.shardedGuilds - chunk
+  const values: number[] = [1000, 9, 500];
```
Comment (ayntee) [inline src/api/controllers/misc.ts:33]:
```suggestion
  const values: [number, number, number] = [1000, 9, 500];
```
A tuple is better-suited.

**Comment by Skillz4Killz** [comment]
This approach is not optimal. Please checkthe approach i spoke about in the other pr

**Comment by OSCAR-WOS** [comment]
> This approach is not optimal. Please checkthe approach i spoke about in the other pr

I am not looking to change my method. When I started on the library first thing I noticed was the long ready event, so I fixed it myself. I have learned a lot in the past 5 days with controllers so it doesn't bother me, however if you prefer other new developers that 5 seconds hang time...

**Pull request closed without merging**